eclipse/src/snap.ts

```diff
@@ -296,6 +296,28 @@ export const createSnapshotForNetwork = async (network: Network) => {
             new BN(0)
           ),
         },
+        protocolFeeX: {
+          tokenBNFromBeginning: "0",
```

Comment:
I'm not sure if the parametr `FromBeginning` is adequate for how it is being used with the fee